documentation/modules/auxiliary/admin/sccm/get_naa_credentials.md

```diff
@@ -0,0 +1,145 @@
+## NAA Credential Exploitation
+
+The NAA account is used by some SCCM configurations in the policy deployment process. It does not require many privileges, but 
+in practice is often misconfigured to have excessive privileges.
+
+The account can be retrieved in various ways, many requiring local administrative privileges on an existing host. However,
+it can also be requested by an existing computer account, which by default most user accounts are able to create.
+
+
+## Module usage
+The `admin/dcerpc/samr_computer` module is generally used to first create a computer account, which requires no permissions:

```

Comment:
shouldn't this be `samr_account` ?

Reply:
Ah yes - wasn't merged when I started this, but will update it here; thanks!